### DISCUSSION TITLE
Mac address for different VIPs in Layer-2 mode

### DISCUSSION THREAD

**Discussion by shadyabhi** [kube-vip/kube-vip#1021] [General]
Do all VIPs bind to different `mac addresses` when using `kube-vip`? 

As per my attempt to get answer from code, `kube-vip` might be advertising different mac addresses for different VIPs, please confirm. 

https://github.com/kube-vip/kube-vip/blob/4c70d873816b4405ff830e0e5eac39f8d5c548d4/pkg/manager/instance.go#L346

Context: MetalLB advertises the same `mac address` for multiple IP addresses and I was wondering if `kube-vip` doesn't do this and I can switch to this. https://github.com/metallb/metallb/issues/1901

**Comment by thebsdbox** [comment]
This code path is only for DHCP load balancer vips. For general service type `LoadBalancer` they will use the MAC address of the interface specified.
